README.md

```diff
@@ -208,7 +208,33 @@ tox -e type
 tox -e type -- reconcile/utils/slack_api.py
 ```
 
+## Run reconcile loop for an integration locally in a container
 
+ This is currently only tested with the docker container engine.
+
+### Prepare config.toml
+
+Make sure the file  [`./config.dev.toml`](https://vault.devshift.net/ui/vault/secrets/app-sre/show/creds/app-interface-dev-config-toml) exists and contains your current configuration.
```

Comment:
I'd rather not put private vault paths in public documentation. They won't help anyone trying to test q-r for any other purposes. This is probably best inside our own repos.

How about:

```
Make sure there is a ./config.dev.toml` file and that its pointing to the following qontract-server address:
...
```

Reply:
I fully agree. Will be more conscious in the future about that stuff :+1: 

Fixed